Source/Core/Common/SysConf.cpp

```diff
@@ -177,8 +187,10 @@ bool SysConf::Save() const
       const u8 data_size = static_cast<u8>(item.bytes.size());
       AppendToBuffer<u8>(&entries, data_size);
       entries.insert(entries.end(), item.bytes.cbegin(), item.bytes.cbegin() + data_size);
-      // Unused byte.
-      entries.insert(entries.end(), '\0');
+      if (item.name == "IPL.NIK")
```

Comment:
Looks like 0x14 is the age restriction (ie. no restriction; 18+ would be 0x12, 16+ would be 0x10 etc.)
Guess thats a custom struct stored in something thats called "small array".